modules/admin/proc-configuring-tls-with-postgresql.adoc

```diff
@@ -0,0 +1,154 @@
+        repo1-retention-full: "1"
+        repo1-retention-full-type: count
+      repoHost:
+        resources:
+          limits:
+            cpu: 300m
+          requests:
+            cpu: 200m
+      repos:
+        - name: repo1
+          schedules:
+            # Every sunday at 01:00 full backup
+            full: "0 1 * * 0"
+            # Monday through saturday at 01:00 differential backup
+            differential: "0 1 * * 1-6"
+          volume:
+            volumeClaimSpec:
+              accessModes:
+                - "ReadWriteOnce"
+              resources:
+                requests:
+                  storage: 2Gi
+      sidecars:
+        pgbackrest:
+          resources:
+            limits:
+              cpu: 300m
+            requests:
+              cpu: 200m
+        pgbackrestConfig:
+          resources:
+            limits:
+              cpu: 300m
+            requests:
+              cpu: 200m
+  users:
+    - name: janus-idp
+      options: "SUPERUSER"
+    - name: sonarqube
+      databases:
+        - sonarqube
+      options: "NOSUPERUSER"
+# ...
+----
+
+. Locate the self-signed certificates in the `postgress-external-db-cluster-cert` secret within the same namespace.
++
+[NOTE]
+====
+You can retrieve the password, hostname, port, and other relevant information for the user `janus-idp` from the `postgress-external-db-pguser-janus-idp` secret.
+====
+
+. Apply the `postgress-external-db-cluster-cert` secret to the namespace where you have deployed your {product-very-short} instance.
+. Update the `values.yaml` Helm configuration file to configure the TLS details as shown in the following example:
++
+[source,yaml]
+----
+global:
+  dynamic:
+    plugins:
+  # ...
+          connection:  # configure Backstage DB connection parameters
+            host: ${POSTGRES_HOST}
+            port: ${POSTGRES_PORT}
+            user: ${POSTGRES_USER}
+            password: ${POSTGRES_PASSWORD}
+            ssl:
+              rejectUnauthorized: true,
+              ca:
+                $file: /opt/app-root/src/postgres-ca.pem
+              key:
+                $file: /opt/app-root/src/postgres-key.key
+              cert:
+                $file: /opt/app-root/src/postgres-crt.pem
+         # ...
+    extraVolumeMounts:
+      # The initContainer below will install dynamic plugins in this volume mount.
+      - mountPath: /opt/app-root/src/postgres-crt.pem
+        name: postgress-external-db-cluster-cert
+        subPath: tls.crt
+      - mountPath: /opt/app-root/src/postgres-ca.pem
+        name: postgress-external-db-cluster-cert
+        subPath: ca.crt
+      - mountPath: /opt/app-root/src/postgres-key.key
+        name: postgress-external-db-cluster-cert
+        subPath: tls.key
+    extraVolumes:
+      # ...
+      - name: postgress-external-db-cluster-cert
+        secret:
+          secretName: postgress-external-db-cluster-cert        
```

Comment:
Unless I missed something it is not that clear what is postgress-external-db-cluster-cert